### DISCUSSION TITLE
How to change color of hinted text?

### DISCUSSION THREAD

**Discussion by gssj85** [AtomMaterialUI/material-theme-issues#531] [Q&A] [answered]
How to change this color please? I'm not sure how it started to show the accent color...it's too bright I can't see where the cursor is:

![Screenshot from 2024-07-12 04-58-50](https://github.com/user-attachments/assets/9cdec85e-256d-4b54-a879-6e408fb7d639)

**Comment by mallowigi** [comment, marked as answer]
Color Scheme > Identifier under caret
